### PR TITLE
Testing part 1

### PR DESCRIPTION
**Pull request by Juank628**
## To Do List: Testing for Add and Remove methods
In this pull request we did the following:
- Installed Jest and jest-environment-jsdom and Babel packages
- Configure Babel and Jest
- Created an addAndRemove test file to test Add and Remove methods

### CODE REVIEW

File: tests/mocks/fileMock.js [hunk 1]
```diff
@@ -0,0 +1 @@
+export default '';
```
Comment (ichala) [inline tests/mocks/fileMock.js:1]: - [ ] to make your repo clean, Kindly remove this empty file since you are not using it anywhere 🙏

**Review by ichala — CHANGES_REQUESTED** [review]
Hi @Juank628 
Good job so far! 
👏👏👏 
You still need to work on some issues to go to the next project, but you're almost there!

the-simpsons-homer-simpson

## To Highlight 🎉

- Add & Remove are tested  ✔️ 
- All tests are green ✔️ 


 ## Required Changes ♻️

- [ ] **_🛑  Check The Comments Below_**





## Optional suggestions

- N/A

**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: tests/mocks/fileMock.js (hunk 1 above)
Comment (Juank628) [reply tests/mocks/fileMock.js:1]: Hi @ichala , thanks for your comment. This file is used as a Mock. We're using that file in the Jest configuration inside the "packaga.json". It's needed to process the images of the application by Jest. Jest doesn't understand images, so this mock replaces images with an empty string.
We are using that file in the following lines of the package.json:
`
"jest": {
    "moduleNameMapper": {
      "\\.(jpg|ico|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$": "<rootDir>/tests/mocks/fileMock.js",
      "\\.(css|less)$": "<rootDir>/tests/mocks/fileMock.js"
    }
`

**Review by mukhammadrizooff — APPROVED** [review]
Hi @TEAM, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- No linter errors ✔️ 
- Good implementations ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: tests/mocks/fileMock.js (hunk 1 above)
Comment (ichala) [reply tests/mocks/fileMock.js:4]: Well its now clear what it does with your awesome comment 💯

**Pull request merged**